src/main/java/org/jabref/gui/sidepane/GroupsSidePaneView.java

```diff
@@ -0,0 +1,56 @@
+package org.jabref.gui.sidepane;
+
+import javafx.scene.control.Button;
+
+import org.jabref.gui.DialogService;
+import org.jabref.gui.actions.SimpleCommand;
+import org.jabref.gui.groups.GroupModeViewModel;
+import org.jabref.gui.groups.GroupViewMode;
+import org.jabref.gui.icon.IconTheme;
+import org.jabref.logic.l10n.Localization;
+import org.jabref.preferences.PreferencesService;
+
+public class GroupsSidePaneView extends SidePaneView {
```

Comment:
Again, just a personal opinion: to distinguish this from the SidePaneContainer i would bring back the term component into these classnames.